src/ui-presets/playback.js

```diff
@@ -81,8 +79,11 @@ class PlaybackUI extends Component {
                   <PictureInPictureOverlay />
                   <PlaybackControls className={style.centerPlaybackControls} />
                   <PlaylistNextScreen />
-                  <InteractiveArea />
+                  <PrePlaybackPlayOverlay />
```

Comment:
1. need to this like this in live preset as well
2. actually I would move change to a separate PR as this solves another issue